jerry-core/ecma/builtin-objects/ecma-builtin-helpers.h

```diff
@@ -27,25 +27,26 @@
  */
 
 ecma_value_t
-ecma_builtin_helper_object_to_string (const ecma_value_t);
+ecma_builtin_helper_object_to_string (const ecma_value_t this_arg);
 ecma_value_t
-ecma_builtin_helper_get_to_locale_string_at_index (ecma_object_t *, uint32_t);
+ecma_builtin_helper_get_to_locale_string_at_index (ecma_object_t *obj_p, uint32_t index);
 ecma_value_t
-ecma_builtin_helper_object_get_properties (ecma_object_t *, bool);
+ecma_builtin_helper_object_get_properties (ecma_object_t *obj_p, bool only_enumerable_properties);
 ecma_value_t
-ecma_builtin_helper_array_concat_value (ecma_object_t *, uint32_t *, ecma_value_t);
+ecma_builtin_helper_array_concat_value (ecma_object_t *obj_p, uint32_t *length_p, ecma_value_t value);
 uint32_t
-ecma_builtin_helper_array_index_normalize (ecma_number_t, uint32_t);
+ecma_builtin_helper_array_index_normalize (ecma_number_t index, uint32_t length);
 uint32_t
-ecma_builtin_helper_string_index_normalize (ecma_number_t, uint32_t, bool);
+ecma_builtin_helper_string_index_normalize (ecma_number_t index, uint32_t length, bool nan_to_zero);
 ecma_value_t
-ecma_builtin_helper_string_prototype_object_index_of (ecma_value_t, ecma_value_t,
-                                                      ecma_value_t, bool);
+ecma_builtin_helper_string_prototype_object_index_of (ecma_value_t this_arg, ecma_value_t arg1,
+                                                      ecma_value_t arg2, bool first_index);
 bool
-ecma_builtin_helper_string_find_index (ecma_string_t *, ecma_string_t *, bool, ecma_length_t, ecma_length_t *);
+ecma_builtin_helper_string_find_index (ecma_string_t *original_str_p, ecma_string_t *search_str_p, bool first_index,
```

Comment:
As mentioned offline already, this is one of the two (?) styles currently present in the codebase. A consistent formatting of function declarations will be applied across the codebase automatically in the near future.